prov/efa/src/rdm/rxr_msg.c

```diff
@@ -699,6 +699,16 @@ struct rxr_op_entry *rxr_msg_alloc_unexp_rx_entry_for_tagrtm(struct rxr_ep *ep,
 	return rx_entry;
 }
 
+/**
+ * @brief Split a posted rx entry to be consumed by multi-recv buffers.
```

Comment:
It is not clear what

`to be consumed by multi-recv buffers`

means. Maybe rephrase a little.

Essentially, this function make consumer_entry, to use buffer in `posted_entry`. Then update posted_entry.